manifest.yml

```diff
@@ -0,0 +1,34 @@
+name : "AWS_IoT_Device_SDK_for_Embedded_C"
+version: "202009.00"
+description: |- 
+  "The AWS IoT Device SDK for Embedded C is a collection of C source
+  files that can be used in embedded applications to securely connect 
+  to AWS IoT Core and interact with other AWS IoT services on AWS Cloud. \n
+  \n
+  See dependencies for included libraries."
+dependencies:
+  - name: "coreMQTT"
+    version: "1.0.0"
+      type: "git"
+      url: "https://github.com/FreeRTOS/coreMQTT"
+  - name: "coreHTTP"
+    version: "1.0.0"
+    repository:
+      type: "git"
+      url: "https://github.com/FreeRTOS/coreHTTP"
+  - name: "coreJSON" 
+    version: "1.0.0" 
+    repository:
+      type: "git"
+      url: "https://github.com/FreeRTOS/coreJSON"
+  - name: "Device-Shadow-for-AWS-IoT-embedded-sdk" 
+    version: "1.0.0"                                
+    repository:                                     
+      type: "git"                                  
+      url: "https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk"
+  - name: "CMock"                                
```

Comment:
CMock is needed only for unit tests. I dont think we should list it.

Reply:
I had discussion with Rashed earlier and he feels it should be there. By that logic we also need CBMC repo as dependency and Rashed agrees adding that. So we either remove CMock or add CBMC related dependency.